crates/core/src/source.rs

```diff
@@ -140,6 +140,7 @@ pub trait Content: Sized {
   fn get_range(&self, range: Range<usize>) -> &[Self::Underlying];
   fn accept_edit(&mut self, edit: &Edit<Self>) -> InputEdit;
   fn get_text<'a>(&'a self, node: &Node) -> Cow<'a, str>;
+  fn get_source_text<'a>(&'a self) -> Cow<'a, str>;
```

Comment:
`get_source_text` is a serious performance overhead. For example, in napi this will re-encode the text everytime when user call get_position